### PR TITLE
DP-33081: Getting redirected to a different page on mass.gov when trying to edit a page

### PR DESCRIPTION
**Pull request by arthurbaghdas**
**Description:**
Explain the technical implementation of the work done.

**Jira:** (Skip unless you are MA staff)
DP-33081

**To Test:**
- [ ] Add steps to test this feature

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: docroot/themes/custom/mass_theme/templates/misc/status-messages.html.twig [hunk 1]
```diff
@@ -21,8 +21,9 @@
  *   - class: HTML classes.
  */
 #}
-
+{{ attach_library('seven/classy.messages') }}
```
Comment (weitzman) [inline docroot/themes/custom/mass_theme/templates/misc/status-messages.html.twig:24]: Should mass_theme now be dependant somehow on seven theme? Does Olivero do that?

**Review by weitzman — COMMENTED** [review]
We can remove  docroot/modules/custom/mass_admin_toolbar/src/Plugin/Block/MessageTabs.php

File: docroot/themes/custom/mass_theme/templates/misc/status-messages.html.twig (hunk 1 above)
Comment (arthurbaghdas) [reply docroot/themes/custom/mass_theme/templates/misc/status-messages.html.twig:24]: We had that dependency when we switched to drupal 10, that's why we brought seven theme as a contrib to our D10

Comment (weitzman) [reply docroot/themes/custom/mass_theme/templates/misc/status-messages.html.twig:24]: OK, I guess we can keep it as an undeclared dependency.

**Pull request closed without merging**